ppr-ui/src/utils/date-helper.ts

```diff
@@ -75,26 +84,38 @@ export function isInt (intValue) {
 }
 
 /**
- * Creates and returns a new Date object in UTC, given parameters in Pacific timezone.
```

Comment:
This description is inaccurate. When you create a date object it is set to user local time zone not UTC.

I updated the function name, and implementation to reflect its purposes. 

All it does is given parameters in Pacific Time it will adjust accordingly to create a date object that matches that time.